time stamp
float32 bid
multi_drone_sar/Task task